### PR TITLE
PRS14027ChristianBarth

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New record and minor corrections in another one.

### CODE REVIEW

File: new/PRS14027ChristianBarth.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2023-05-26">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1" sameAs="wd:Q84831">
+                    <persName xml:lang="gez" xml:id="n1">Christian Gottlob Barth</persName>
```
Comment (eu-genia) [inline new/PRS14027ChristianBarth.xml:48]: lang=de

File: new/PRS14027ChristianBarth.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
```
Comment (eu-genia) [inline new/PRS14027ChristianBarth.xml:49]: delete line

File: new/PRS14027ChristianBarth.xml (hunk 1 above)
Comment (eu-genia) [reply new/PRS14027ChristianBarth.xml:48]: `<persName xml:lang="de" xml:id="n1"><forename>Christian</forename> <forename>Gottlob</forename> <surname>Barth</surname></persName>`

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS14028HabtaMikael.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14028HabtaMikael" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Habta Mikael</title>
```
Comment (thea-m) [inline new/PRS14028HabtaMikael.xml:8]: transcription

**Review by thea-m — CHANGES_REQUESTED** [review]

File: new/PRS14027ChristianBarth.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/PRS14027ChristianBarth.xml:48]: ok

File: new/PRS14028HabtaMikael.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/PRS14028HabtaMikael.xml]: ok

**Review by eu-genia — APPROVED** [review]

**Pull request merged**